libknet/libknet.h

```diff
@@ -1223,7 +1222,6 @@ int knet_link_get_status(knet_handle_t knet_h, uint16_t host_id, uint8_t link_id
 #define KNET_SUB_PMTUD_T     10 /* Path MTU Discovery thread */
 #define KNET_SUB_TRANSPORT_T 11 /* Transport common */
 #define KNET_SUB_UDP_LINK_T  12 /* UDP Transport */
-#define KNET_SUB_TCP_LINK_T  13 /* TCP Transport */
```

Comment:
Leave the hole here. We need to re-arrange all logging codes to leave space for growth anyway.